sdk/cognitiveservices/cognitiveservices-inkrecognizer/test/utils/testUtils.ts

```diff
@@ -0,0 +1,29 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export const isNode =
+  !!process && !!process.version && !!process.versions && !!process.versions.node;
+
+export enum EnvVarKeys {
+  INKRECOGNIZER_API_KEY = "INKRECOGNIZER_API_KEY"
```

Comment:
This model of storing env var names made sense in Service Bus because we were dealing with multiple env vars.
Here, we only have one.

I would suggest to forego this model and have the test case directly call either `process.env` or `window.__env__` based on whether the test is being run in node or browser.

Reply:
done